package.json

```diff
@@ -44,7 +44,7 @@
     "release:prepare:build:es6": "npm run release:prepare:build:es6:copy && npm run release:prepare:build:es6:replace",
     "release:prepare:build:es6:copy": "cp -R -f ./src/*.src.js ./vl-datepicker.js",
     "release:prepare:build:es6:replace": "npm run release:prepare:build:es6:replace:css && npm run release:prepare:build:es6:replace:dist && npm run release:prepare:build:es6:replace:src",
-    "release:prepare:build:es6:replace:css": "replace \"'/style.css'\" \"'/node_modules/vl-ui-datepicker/style.css'\" vl-datepicker.js",
+    "release:prepare:build:es6:replace:css": "replace \"('|\\\")/style.css('|\\\")\" \"'/node_modules/vl-ui-datepicker/style.css'\" vl-datepicker.js",
```

Comment:
`['\\\"]` is normaal gezien equivalent (gebruiken we in scripts hieronder al)

Reply:
Perfect, gaat sowieso verplaatst worden naar vl-ui-util, dus heb aan @pascalriquier gevraagd dit op te pikken. Hij kan de variant dan kiezen.